patches/minecraft/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -276,6 +280,15 @@
                          this.m_21008_(InteractionHand.MAIN_HAND, ItemStack.f_41583_);
                       }
                    }
+@@ -1212,7 +_,7 @@
+                   }
+                }
+             }
+-
++            this.m_36334_();
```

Comment:
A small comment should be added summarizing what this patch does: `//FORGE: whatever`.
Also, small nitpick: don't remove the empty line, just add the call in a separate line above it.